defaults.yaml

```diff
@@ -68,3 +68,7 @@ loom:
 # asset server (must NOT be localhost because it's called from within container)
 assets:
   url: "http://localhost:3070"
+
+# auth credentials service (i.e., for stash)
+auth:
```

Comment:
Can this work with anything other than stash? If not shouldn't this be called stash config? Maybe grouped under the configuration for the stash plugin specifically?